localMatchApp/sources/Login.cpp

```diff
@@ -0,0 +1,82 @@
+#include "Login.h"
+
+using namespace std;
+
+Login::Login ( ) {
+  // Connect to database.
+}
+
+Login::~Login ( ) {
+  // Disconnect to database.
+}
+
+Login::connectionMessage Login::signUp ( const string i_user, const string i_password,
+                                         const string i_repeatedPassword ) {
+  // Send information to database.
+  string userInDatabase = "";
+
+  if ( hasCorrectFormat ( i_password ) && hasCorrectFormat ( i_repeatedPassword ) ) {
+    if ( i_user.compare ( userInDatabase ) == 0 ) {
+      if ( i_password == i_repeatedPassword ) {
+        // Set user and password in table
+        return connectionMessage::goodEntry;
+      } else {
+        return connectionMessage::notEqualPasswords;
+      }
+    } else {
+      return connectionMessage::ocupatedUser;
+    }
+  } else {
+    return connectionMessage::wrongPassword;
+  }
+}
+
+Login::connectionMessage Login::entry ( const string i_user, const string i_password ) {
+  // Search the user and compare password
+  // Query asking for password based on user
+  string userInDatabase = "";
+  string passInDatabase = "";
+
+  if ( i_user != userInDatabase ) {
+    return connectionMessage::doesntExists;
+  } else if ( i_password != passInDatabase ) {
+    return connectionMessage::wrongPassword;
+  } else {
+    return connectionMessage::goodEntry;
+  }
+}
+
+bool Login::hasACapital ( const string i_password ) {
+  for ( auto &i : i_password ) {
+    if ( isupper ( i ) ) {
+      return true;
```

Comment:
Same as before, remove multiple returns when possible. Check the rest of your code